flow/templates/base_script.sh

```diff
@@ -18,6 +18,8 @@ cd {{ project.config.project_dir }}
 {% for operation in operations %}
 
 # {{ "%s"|format(operation) }}
+{% block pre_operation %}
```

Comment:
This is missing the `operation` variable from the upper scope when it is overridden in `stampede2.sh`. Defining this as a `scoped` block should help.
https://jinja.palletsprojects.com/en/2.11.x/templates/#block-nesting-and-scope
```suggestion
{% block pre_operation scoped %}
```